vpr/src/base/read_options.cpp

```diff
@@ -1474,6 +1474,12 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .default_value("off")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    gen_grp.add_argument<bool, ParseOnOff>(args.fail_status_if_fail_timing, "--fail-status-if-fail-timing")
```

Comment:
I'd rename this.  
1. We use _ to separate words in command line options (vpr convention) so we should stick with that.
2. terminate_if_timing_fails would be a better name, as it indicates what happens if this is "on".